internal/mock/sqladmin.go

```diff
@@ -229,3 +249,40 @@ func CreateEphemeralSuccess(i FakeCSQLInstance, ct int) *Request {
 	}
 	return r
 }
+
+// CreateEphemeralSuccess returns a Request that responds to the
+// `sslCerts.createEphemeral` SQL Admin endpoint. It responds with a "StatusOK" and a
+// SslCerts object.
+//
+// https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/sslCerts/createEphemeral
+func CreateEphemeralSuccess(i FakeCSQLInstance, ct int) *Request {
+	return CreateEphemeralSuccessWithExpiry(i, ct, time.Now().Add(time.Hour))
+}
+
+// TestClient is a convenience wrapper that configures a mock HTTP client and
+// sqladmin.Service based on a connection name.
+func TestClient(proj, reg, name string, db *sqladmin.DatabaseInstance, certExpiry time.Time) (*sqladmin.Service, func() error, error) {
+	inst := NewFakeCSQLInstance(proj, reg, name)
+	mc, url, cleanup := HTTPClient(
+		InstanceGetSuccessWithDatabase(inst, 1, db),
+		CreateEphemeralSuccessWithExpiry(inst, 1, certExpiry),
```

Comment:
I don't really like the way we've pushed this into another function. It makes it less clear from each tests perspective what's being blocked.